features/step_definitions/networking.rb

```diff
@@ -886,4 +886,23 @@
   @result = host.exec_admin("ifconfig #{cb.interface}")
   cb[cb_ipaddr]=@result[:response].match(/\d{1,3}\.\d{1,3}.\d{1,3}.\d{1,3}/)[0]
   logger.info "The Internal IP of node is stored in the #{cb_ipaddr} clipboard."
-end
\ No newline at end of file
+end
+
+Given /^I store "([^"]*)" node's corresponding sdn or ovnkube pod name in the#{OPT_SYM} clipboard$/ do |node_name,cb_pod_name|
```

Comment:
This step def will be savior in future for all sdn and ovn tests

Reply:
@anuragthehatter IMO, this step name should be something like `I store ...node's default network type pod name...`

Reply:
Commited! Smart suggestion @pruan-rht , thanks